test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java

```diff
@@ -376,11 +405,62 @@ protected static void checkMockedIngestionHealth() throws Exception {
 		assertEquals(MockedAppInsightsIngestionServlet.PONG, postResponse);
 	}
 
-	protected static void startDockerContainer() throws Exception {
+	private static void createDockerNetwork() throws Exception {
+		try {
+			networkId = docker.createNetwork(networkName);
+		} catch (Exception e) {
```

Comment:
This only throws IOException and InterruptedException. Please make Exception type specific.

Reply:
It's common practice to throw Exception in test code. Everything written thus far follows that convention. I'd only catch something specific when a specific action based on exception type is needed.

Reply:
Same as above. It might be common within our team / or group, but when people want to contribute over on GitHub they might come from different work place and different culture. Clear instructions would help them and save our time too.